tests/baselines/reference/ambientDeclarations.js

```diff
@@ -77,7 +77,19 @@ declare module 'external1' {
 
 
 //// [ambientDeclarations.js]
+// Ambient variable with type annotation
+// Ambient function with no type annotations
+// Ambient function with type annotations
+// Ambient function with valid overloads
+// Ambient function with optional parameters
+// Ambient class
+// Ambient enum
+// Ambient enum with integer literal initializer
+// Ambient enum members are always exported with or without export keyword

```

Comment:
But issue #2546 is precisely about keeping these comments. And it said PRs are welcomed. Can we define what are precisely the conditions to hide or show comments? Should we keep all comments of the type `/** */` or also `//` (the original reporter in #2546 wanted the latter too).